repository/g.json

```
@@ -1590,6 +1590,18 @@
 				}
 			]
 		},
+		{
+			"name": "GoogleSQL Syntax Highlighting",
+			"details": "https://github.com/pratik-m/sublime-googlesql-syntax",
+			"author": "Pratik Munot",
+			"labels": ["syntax", "GoogleSQL", "BigQuery"],
```

```suggestion
			"labels": ["language syntax", "GoogleSQL", "BigQuery"],
```

https://packagecontrol.io/browse/labels/language%20syntax